src/main/java/com/thealgorithms/others/ArrayRightRotation.java

```diff
@@ -0,0 +1,32 @@
+package com.thealgorithms.others;
+
+/*
+ * A right rotation operation on an array
+ * shifts each of the array's elements
+ * given integer n unit to the right.
+ *
+ * @author sangin-lee
+ */
+
+public class ArrayLeftRotation {
```

Comment:
It can be a _proper utility class_.
```suggestion
public final class ArrayLeftRotation {
    private ArrayLeftRotation() {
    }
```